pydantic_ai_slim/pyproject.toml

```diff
@@ -55,6 +55,8 @@ dependencies = [
     "exceptiongroup; python_version < '3.11'",
     "opentelemetry-api>=1.28.0",
     "typing-inspection>=0.4.0",
+    "tenacity>=9.1.2",
+    "throttled-py>=2.2.0"
```

Comment:
@Kludex What do you think of these being included by default? Should we put them in an optional dependency group?